### PR TITLE
Create DocClass table

### LINKED ISSUE #77: Add DocumentClass table

### PR DESCRIPTION
**Pull request by chyku**
Resolves #77

<Brief description of the added features or other changes/updates (either bulletpointed list, small paragraph, etc.)>

### CODE REVIEW

**Comment by rgychiu** [comment]
I wish I could react to commit messages

**Comment by rgychiu** [comment]
WE'RE NUTS

File: backend/api/views/docclass.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+@docclass.route("/document_class/new", methods=["POST"])
+def add_document_class():
+    """ function that is called when you visit /document_class/new, creates a new docclass """
+    data = request.get_json()
+    logger.info(data)
+    if "name" not in data:
+        return create_response(
+            status=422, message="No name provided for new Document Class"
```
Comment (arpanlaha) [inline backend/api/views/docclass.py]: should be 400

File: backend/tests/test_document_class.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    DocumentClass.query.delete()
+    db.session.commit()
+
+    # Test for not having required field provided
+    rs = client.post(
+        "/document_class/new",
+        content_type="application/json",
+        json={"description": "description here"},
+    )
+    assert rs.status_code == 422
```
Comment (arpanlaha) [inline backend/tests/test_document_class.py]: 400

File: backend/api/models/DocumentClass.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# Note that we use sqlite for our tests, so you can't use Postgres Arrays
+class DocumentClass(Mixin, db.Model):
+    """Document Class Table."""
+
+    __tablename__ = "document_class"
+
+    id = db.Column(db.String, unique=True, primary_key=True)
+    name = db.Column(db.String, unique=True)
+    description = db.Column(db.String, unique=False, nullable=True)
```
Comment (rgychiu) [inline backend/api/models/DocumentClass.py]: Good use of constraints on this table

File: backend/api/views/docclass.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+from api.core import create_response, serialize_list, logger
+
+import requests, json
+
+docclass = Blueprint("docclass", __name__)
+
+
+@docclass.route("/document_class", methods=["GET"])
+def get_document_class():
+    """ function that is called when you visit /document_class, gets all the docclasses """
```
Comment (rgychiu) [inline backend/api/views/docclass.py]: Good documentation with docstrings

**Pull request merged**